spark-jobs/src/main/scala/za/co/absa/enceladus/common/SparkCompatibility.scala

```diff
@@ -26,6 +26,6 @@ import za.co.absa.commons.version.Version._
 import za.co.absa.commons.version.impl.SemVer20Impl.SemanticVersion
 
 object SparkCompatibility {
-  val minSparkVersionIncluded: SemanticVersion = semver"2.4.2"
+  val minSparkVersionIncluded: SemanticVersion = semver"3.0.0"
```

Comment:
Considering the known problems with lower versions, what about setting this to 3.1.2?

Reply:
It does not make sense to me for this version, since it is spark3.0-based. we can do this in the spar3.1 version or if we merge the spark3.1 into `develop-spark-3`. Does this make sense to you?

Reply:
But that (#1755) has been already merged. 😉

Reply:
Oh, I didn't notice that. In that case, you are completely correct and I will make the adjustment as suggested 👍

Reply:
Done.